packages/react/src/transactions/adapters/__tests__/FollowProfilesCallGateway.spec.ts

```diff
@@ -153,14 +201,111 @@ describe(`Given an instance of the ${FollowProfilesCallGateway.name}`, () => {
           data: mockCreateFollowTypedDataData({ nonce }),
         }),
       ]);
-      const followFeeTransactionGateway = new FollowProfilesCallGateway(apollo);
+      const mockTransactionObserver = mockITransactionObserver();
+      const factory = mockITransactionFactory(mockTransactionObserver);
+      const followProfilesCallGateway = mockFollowProfilesCallGateway({ apollo, factory });
 
-      const unsignedCall = await followFeeTransactionGateway.createUnsignedProtocolCall(
+      const unsignedCall = await followProfilesCallGateway.createUnsignedProtocolCall(
         request,
         nonce,
       );
 
       expect(unsignedCall.nonce).toEqual(nonce);
     });
   });
+
+  describe(`when calling the "${FollowProfilesCallGateway.prototype.createProxyTransaction.name}" method`, () => {
+    describe('with an UnconstrainedFollowRequest', () => {
+      const request = mockUnconstrainedFollowRequest();
+      describe('and receiving a successful response', () => {
+        it(`should resolve with ${ProxyTransaction.name} on Polygon`, async () => {
+          const indexingId = 'indexing-id';
+          const apollo = createMockApolloClientWithMultipleResponses([
+            createBroadcastProxyActionCallMockedResponse({
+              result: indexingId,
+              variables: {
+                request: {
+                  follow: {
+                    freeFollow: {
+                      profileId: request.profileId,
+                    },
+                  },
+                },
+              },
+            }),
+          ]);
+          const mockTransactionObserver = mockITransactionObserver();
+          const factory = mockITransactionFactory(mockTransactionObserver);
+          const followProfilesCallGateway = mockFollowProfilesCallGateway({ apollo, factory });
+
+          const transactionResult = await followProfilesCallGateway.createProxyTransaction(request);
+
+          if (transactionResult.isFailure()) throw transactionResult.error;
+
+          const transaction = transactionResult.value;
+
+          await transaction.waitNextEvent();
+
+          expect(transaction).toEqual(
+            expect.objectContaining({
+              chainType: ChainType.POLYGON,
+              id: 'id',
+              request,
+              status: ProxyActionStatus.MINTING,
+            }),
+          );
+
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+          expect((transaction as any).state.proxyId).toEqual(indexingId);
+        });
+      });
+
+      describe('and receiving an error response', () => {
+        it.only('should return broadcast error with fallback', async () => {
```

Comment:
`.only` + similar considerations done in collect gateway regarding tests organization/wording.